apps/explorer/lib/explorer/generic_paging_options.ex

```diff
@@ -0,0 +1,19 @@
+defmodule Explorer.GenericPagingOptions do
```

Comment:
This is possibly the wrong location for this file, as it seems this is all to do with visual pagination display - and despite being named "Generic" is just for one instance of pagination.

Reply:
I copied it from `apps/explorer/lib/explorer/paging_options.ex`. 

As for `Generic` the one I mentioned wasn't... well... generic enough to serve a common pagination use case (where you can sort by different fields, define page size etc).